config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_vertexaidatasets.vertexai.cnrm.cloud.google.com.yaml

```diff
@@ -67,11 +67,215 @@ spec:
+                    type: string
+                  namespace:
+                    description: 'Namespace of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/'
+                    type: string
+                type: object
+              region:
+                description: Immutable. The region of the dataset. eg us-central1.
+                type: string
+              resourceID:
+                description: Immutable. Optional. The service-generated name of the
+                  resource. Used for acquisition only. Leave unset to create a new
+                  resource.
+                type: string
+            required:
+            - displayName
+            - metadataSchemaUri
+            - projectRef
+            type: object
+          status:
+            properties:
+              conditions:
+                description: Conditions represent the latest available observation
+                  of the resource's current state.
+                items:
+                  properties:
+                    lastTransitionTime:
+                      description: Last time the condition transitioned from one status
+                        to another.
+                      type: string
+                    message:
+                      description: Human-readable message indicating details about
+                        last transition.
+                      type: string
+                    reason:
+                      description: Unique, one-word, CamelCase reason for the condition's
+                        last transition.
+                      type: string
+                    status:
+                      description: Status is the status of the condition. Can be True,
+                        False, Unknown.
+                      type: string
+                    type:
+                      description: Type is the type of the condition.
+                      type: string
+                  type: object
+                type: array
+              createTime:
+                description: The timestamp of when the dataset was created in RFC3339
+                  UTC "Zulu" format, with nanosecond resolution and up to nine fractional
+                  digits.
+                type: string
+              name:
+                description: The resource name of the Dataset. This value is set by
+                  Google.
+                type: string
+              observedGeneration:
+                description: ObservedGeneration is the generation of the resource
+                  that was most recently observed by the Config Connector controller.
+                  If this is equal to metadata.generation, then that means that the
+                  current reported status reflects the most recent desired state of
+                  the resource.
+                type: integer
+              updateTime:
```

Comment:
I propose ignoring this field for now; I think it will likely be out of date whenever a change is made externally.